### PR TITLE
Sass 1284

### PR DESCRIPTION
**Pull request by HarryGull**
### Description
Create 'are you still working at employer' radio button EOY page.

[SASS-1236](https://jira.tools.tax.service.gov.uk/browse/SASS-1236)

##### Before Reviewing
- [x]  Have you pulled the branch down?
- [x]  Have you assigned yourself to the PR?
- [x]  Have you moved the task to “in review” on JIRA?
- [x]  Have you checked to ensure all dependencies are up to date?
- [x]  Have you checked to ensure its been rebased against the current version of master?

##### Whilst Reviewing
- [x]  Have you run the tests?
- [x]  Have you run the journey tests?
- [x]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?

##### After Reviewing
- [x]  Have you checked for merge conflicts or any changes in the current master that may affect the current pull request? i.e. does it need another rebase?
- [x]  Have you checked to make sure there are no builds in the pipeline before you merge?
- [x]  Have you moved the task to “in pipeline” on Jira?

##### Before creating PR
- [x]  Have you run the tests?
- [x]  Have you run the journey tests? (where applicable)
- [x]  Have you addressed warnings where appropriate?
- [x]  Have you rebased against the current version of master?
- [x]  Have you checked code coverage isn’t lower than previously?

##### After PRs been raised
- [ ]  Have you checked the PR Builder passes?

### CODE REVIEW

File: app/views/employment/StillWorkingForEmployerView.scala.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+@layout(pageTitle = titleForUse, taxYear = Some(taxYear), hasErrors = hasFormErrors, isAgent = user.isAgent) {
+
+@errorSummary(form.errors.distinct)
+
+    @formWithCsrf(action = controllers.employment.routes.StillWorkingForEmployerController.submit(taxYear, employmentId)) {
+        @inputRadio(
+            form = form,
+            heading = headingForUse,
+            content = None,
```
Comment (ina-tu) [inline app/views/employment/StillWorkingForEmployerView.scala.html:47]: `content = None` can be removed. It's set to none in the input radio helper

**Review by ina-tu — CHANGES_REQUESTED** [review]

File: test/services/RedirectServiceSpec.scala [hunk 2]
```diff
@@ -80,34 +88,38 @@ class RedirectServiceSpec extends UnitTest {
     "redirect to check employment details page when no cessation date question" in {
 
       val response = RedirectService.employmentDetailsRedirect(cyaModel.copy(cyaModel.employmentDetails.copy(
-        employerRef = Some("123/12345"), startDate = Some("2020-11-01"), taxablePayToDate = Some(1), totalTaxToDate = Some(1), payrollId = Some("id")
-      )),taxYear,"employmentId",false)
+        employerRef = Some("123/12345"), startDate = Some("2020-11-01"), cessationDateQuestion = Some(true), taxablePayToDate = Some(1),
```
[the remark this answers is not in the document]
Comment (matt712) [reply test/services/RedirectServiceSpec.scala:91]: It's listed in the employmentDetails of the CYA model

File: app/views/employment/CheckEmploymentDetailsView.scala.html [hunk 3]
```diff
@@ -68,7 +68,7 @@
 
         val stillWorkingForEmployer = EmploymentDataForView(messages(s"$msgKeyGeneric.stillWorkingForEmployer"),
             if(employmentDetails.cessationDate.isDefined) {Some(messages("common.no"))} else {
-                if(employmentDetails.cessationDateQuestion.contains(true)){Some(messages("common.yes"))} else {Some(messages("common.no"))}},
+                if(employmentDetails.cessationDateQuestion.contains(true)){Some(messages("common.no"))} else {Some(messages("common.yes"))}},
```
Comment (ina-tu) [inline app/views/employment/CheckEmploymentDetailsView.scala.html:71]: I think the yes and no should be the other way around. When I clicked yes on the question form it displays as no on cya page

**Review by ina-tu — CHANGES_REQUESTED** [review]

**Review by ina-tu — APPROVED** [review]

**Comment by ina-tu** [comment]
waiting for journey tests to be reviewed

**Pull request merged**